.changeset/brown-carpets-obey.md

```diff
@@ -0,0 +1,7 @@
+---
+'zod-validation-error': minor
+---
+
+Match `ZodError` via heuristics instead of relying on `instanceof`.
+
+_Why?_ Because we want to ensure that zod-validation-error works smoothly even when multiple version of zod have been installed in the same project.
```

Comment:
[suggestion]

```suggestion
_Why?_ Because we want to ensure that zod-validation-error works smoothly even when multiple versions of zod have been installed in the same project.
```

Reply:
Thanks for pointing out this typo. Fixed.